Lavinia-api/Data/CustomValidation.cs

```diff
@@ -64,12 +107,25 @@ private static void ValidateCounty(County county, HashSet<int> checkedModels)
         /// <param name="checkedModels">A set of models that have already been checked.</param>
         public static void ValidateCountyData(CountyData countyData, HashSet<int> checkedModels)
         {
-            if (countyData == null) throw new ArgumentException("CountyData cannot be null.");
+            if (countyData == null)
+            {
+                throw new ArgumentException("CountyData cannot be null.");
+            }
+
+            if (countyData.Year == -1)
+            {
+                throw new ArgumentException("CountyData.Year cannot be the default value.");
+            }
 
-            if (countyData.Year == -1) throw new ArgumentException("CountyData.Year cannot be the default value.");
             if (countyData.Population == -1)
+            {
                 throw new ArgumentException("CountyData.Population cannot be the default value.");
```

Comment:
Changing text in an exception is a refactor, as well as unrelated to the commit's message.

Reply:
The text in the exception on this line wasn't changed.

Reply:
My bad, then!